amplitude/context_plugin.go

```diff
@@ -36,7 +40,9 @@ func (c *ContextPlugin) Execute(event *Event) *Event {
 		event.InsertID = uuid.NewString()
 	}
 
-	event.Library = c.contextString
+	event.Library = p.contextString
 
 	return event
 }
+
+var _ EnrichmentPlugin = (*ContextPlugin)(nil)
```

Comment:
Curious what is this line doing?

Reply:
Compile-time check that `ContextPlugin` implements `EnrichmentPlugin` interface. Unnecessary now (in `v0.0.4`) - removed.

Reply:
Got it, thanks for the explanation!